src/components/SingleQuote.astro

```diff
@@ -21,10 +22,13 @@ const { classString, testimonial } = Astro.props as Props;
 >
   <div class="section-content bg-navy mb-16 p-4 rounded-md lg:mb-20 xl:py-20">
     <figure class="flex flex-col p-4 relative text-white xl:px-16 xl:py-0">
-      <img
-        class="h-8 max-w-full mb-3 self-start w-auto"
-        src={testimonial.svgSrc}
-      />
+      <h2>
```

Comment:
Is wrapping in an `h2` something that is recommended somewhere? Seems a bit odd to wrap the `img` tag.

Reply:
I think you're right - I was thinking of the company name displayed here as a header but you probably don't need it for context for the testimonial quote